### DISCUSSION TITLE
[GitHub integration] Secreted environment variables support

### DISCUSSION THREAD

**Discussion by frengooo** [mia-platform/community#152] [Q&A] [answered]
Hi guys,
the Mia-Platform console works with GitHub too as git provider.

Integrating the console with GitHub, how are handled the _secreted environment variables_?
Can be handled directly inside GitHub, as we do with GitLab, or we need an external secret manager?

Thank you 😄

**Comment by Danielecina** [comment, marked as answer]
Hi @silversoul93, currently the secrets environment variables not be handled by existing sections in Console. In the future, it is planned to support it. Today, You can set secrets directly on GitHub like Gitlab. Let me know 👍

**Reply by frengooo** [reply]
Hi @Danielecina, thanks for your reply 😄 probably we will need this feature.
Possibly we will let you know.

**Reply by fredmaggiowski** [reply]
@silversoul93 if you need the feature please add a tracking issue on the internal roadmap 

@albertotessarotto @jacoposte FYI
